src/main/java/net/mcreator/element/types/Biome.java

```diff
@@ -130,9 +140,13 @@ public Biome(ModElement element) {
 	}
 
 	@Override public BufferedImage generateModElementPicture() {
+		int treeType = 0;
+		if(customTree) {
+			treeType = 1;
+		}
 		return MinecraftImageGenerator.Preview

```

Comment:
This part is still missing: and pass treesPerChunk as 0 if nor vanilla nor custom trees are enabled, to disable drawing trees in such case. - this way tree won't be drawn if no tree would be defined but treesPerChunk is non-zero.